libraries/src/AWS.Lambda.Powertools.Tracing/Internal/TracingAspectHandler.cs

```diff
@@ -163,12 +163,25 @@ public T OnException<T>(AspectEventArgs eventArgs, Exception exception)
         if (CaptureError())
         {
             var nameSpace = GetNamespace();
-
+            
+            var message =
```

Comment:
consider using StringBuilder for example
var sb = new StringBuilder();
sb.AppendLine($"Exception type: {exception.GetType()}");
sb.AppendLine($"Exception message: {exception.Message}");
...

message = sb.ToString();